stock-plugins/aiverify.stock.accumulated-local-effect/algorithms/accumulated_local_effect/algo_init.py

```diff
@@ -37,7 +38,7 @@ def progress_callback(completion_value: int):
         Args:
             completion_value (int): Current progress completion
         """
-        print(f"[PluginTest] Progress Update: {completion_value}")
+        print(f"[AlgoInit] Progress Update: {completion_value}")
```

Comment:
Instead of `[AlgoInit]` let's print `[Accumulate Local Effect Test]`? It's probably more meaningful and descriptive.